chpl/chpl-service/src/main/java/gov/healthit/chpl/auth/jwt/CognitoRsaKeyProvider.java

```diff
@@ -0,0 +1,46 @@
+package gov.healthit.chpl.auth.jwt;
+
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.interfaces.RSAPublicKey;
+
+import com.auth0.jwk.JwkException;
+import com.auth0.jwk.JwkProvider;
+import com.auth0.jwk.JwkProviderBuilder;
+import com.auth0.jwt.interfaces.RSAKeyProvider;
+
+public class CognitoRsaKeyProvider implements RSAKeyProvider {
+    private final URL awsKidStoreUrl;
+    private final JwkProvider provider;
+
+    public CognitoRsaKeyProvider(String awsCognitoRegion, String awsUserPoolId) {
+        String url = String.format("https://cognito-idp.%s.amazonaws.com/%s/.well-known/jwks.json", awsCognitoRegion, awsUserPoolId);
```

Comment:
It feels like some of this should be in an environment property or at least a constant